docker/example-rollup.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:jammy
+
+ARG TARGETARCH
+
+RUN apt-get update && apt-get install -y --no-install-recommends libcurl4 curl \
```

Comment:
Let's add `tini` here as well, like in 

https://github.com/EspressoSystems/espresso-sequencer/pull/657/files